src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -207,9 +207,10 @@ private void setUpAlertsTriggers(List<Trigger> triggers) {
         List<Trigger> filteredTriggers = new ArrayList<>();
 
         for (Trigger trigger : triggers) {
-            if (trigger.getId().endsWith(resource.getId())) {
-                filteredTriggers.add(trigger);
-            }
+                if ( trigger.getTags()!=null && trigger.getTags().containsKey("resourceId") && trigger.getTags().get
```

Comment:
Also, it would be better to make it more readable, by splitting the conditions each on one line:

    trigger.getTags()!=null 
    && trigger.getTags().containsKey("resourceId") 
    && trigger.getTags().get("resourceId").equals(resource.getId())

In fact, I believe this could be simplified:

    trigger.getTags()!=null 
    && resource.getId().equals(trigger.getTags().get("resourceId"))

As long as `resource` is never null.